### DISCUSSION TITLE
PhotoSync sync to WebDav do not appear in PhotoPrism Search

### DISCUSSION THREAD

**Discussion by ant0nwax** [photoprism/photoprism#2956] [Q&A]
Hi,

EDIT: THIS FIRST POST IS JUST KEPT FOR REFERENCE, I CHANGED THE TITLE OF THE DISCUSSION TO BE FOCUSING ON PHOTOSYNC

I had an issue when the physical volume / filesystem of the docker-compose vm that i use only for photoprism went full 100% use.
After extending the physical volume, vg, lv, fs i cannot upload/import anymore, but i do not see errors...
most likely that happened shortly after 19th of november

Android Photo Uploader uploads and starts import but no new photos appear,
i see in `docker logs -f` the following after the upload is completed

```
photoprism_1  | time="2022-11-29T10:56:07Z" level=info msg="moving files from import"
photoprism_1  | time="2022-11-29T10:56:07Z" level=info msg="import: found no .ppignore file"
photoprism_1  | time="2022-11-29T10:56:07Z" level=info msg="import: imported 1 file"
photoprism_1  | time="2022-11-29T10:56:11Z" level=info msg="import completed in 4 s"
```

i do not find photos newer than November 19 on the 
```  - "~/Pictures:/photoprism/originals"              ```

import folder is empty

no errors in Photo Uploader and no errors on the docker containers... no on the browser

i simply would like to understand where the photos are uploaded... i could share more info or config, but maybe it is trivial.

Thank you for any  ideas

**Comment by lastzero** [comment]
"Android Photo Uploader" is a third party application we do not support.

**Reply by ant0nwax** [reply]
I understand, that a 3rd party tool is unsupported in this discussion, but i am sure that the issue is not about that app at all because it worked perfect before.

Let's forget the app and look at PhotoSync, that is officially recommended
I uninstalled PhotoUploader and use the PhotoSync now
WebDav is configured and i am able to choose from the following folders in https://admin@photoprism.mydomain.tld/originals
(those years were created by PhotoPrism Uploader and the PHOTOS is a mount that i added from my old photos)
```
2017
2020
2021
2022
PHOTOS
```

For the Sync at PhotoSync i chose the following folder
```
/2022/11
```
and i synced to Device Name + Folder Name
so the missing ~130 photos appeared at 
```
/2022/11/DeveiceName/FolderName/
```
i reconfigured that to 
```
/
```
and moved `/2022/11/DeveiceName/FolderName/` to `/DeveiceName/FolderName/` (inlucding the ~130 photos)
PhotoSync makes a directure structure anyway inside with DeveiceName + FolderName
PhotoPrism Uploader made that structure for /YYYY/MM instead

when i checked photoprism GUI i seen that at Library an indexing of all 3284 photos since the start of using PhotoPrism Uploader started

```
2017
2020
2021
2022
PHOTOS
'Samsung SM-N970F'
```

i will update this post when the Library Indexing is finished, but i think this will take a couple of hours or even days

PS,if I share my experience here I also do not support PhotoPrism Uploader anymore because i changed to PhotoSync

I tried to uninstall and reinstall PhotoPrism Uploader and i just got a black screen app :) current version is broken? anyway not of any interest. So thats where I installed PhotoSync.

Thank you for reading, if you have some ideas what could have happened maybe it means something to you

**Reply by lastzero** [reply]
It looks like the import volume is commented out, so it would not surprise if it's empty?

**Reply by ant0nwax** [reply]
Thank you i changed all my posts, i think it always worked without any import folder, so this is not the topic most likely, please could you explain why Library Indexing restarted for the complete `~/Pictures` folder?

**Reply by lastzero** [reply]
https://docs.photoprism.app/user-guide/library/originals/#automatic-indexing

**Reply by ant0nwax** [reply]
thank you
will it happen each time I upload 1 photo with WebDav?
does it make sense when I upload 130 new photos via WebDav to reindex all files that where placed in the ~/Pictures folder?
with PhotoPrism Uploader this worked different
My Server is overloaded because of the indexing and this cannot be a daily behaviour, the docs do not say that A complete reindexing happens

**Reply by graciousgrey** [reply]
After files have been moved from import to originals a normal rescan is triggered, independent from which sync app you used.  A normal rescan skips already indexed files that have not been changed since the latest indexing.

**Reply by ant0nwax** [reply]
my installation has no import folder as lastzero already mentioned
a ~/Pictures indexing has been triggered by moving  `~/Pictures/2022/11/DeveiceName/FolderName/` to `~/Pictures//DeveiceName/FolderName/` (inlucding the ~130 photos)
i am not yet sure if it a full rescan or it will stop
already indexed files are re-indexed at the moment which is breaking the complete server because it has a load of 4.5 on a 4 core machine

**Reply by graciousgrey** [reply]
After new files have been added to originals using WebDAV only a normal re-indexing is triggered, not a complete one.  I just tested this again. 

If this is too much for your server I would recommend going through our system requirements and performance checklist: 
- https://docs.photoprism.app/getting-started/#system-requirements
- https://docs.photoprism.app/getting-started/troubleshooting/performance/

Also make sure to use mariadb and not sqlite:
- https://docs.photoprism.app/getting-started/troubleshooting/sqlite/

Kindly note: Analyzing your personal setup and logs is not in the scope of our free community support.

**Reply by ant0nwax** [reply]
Sure, i am not expecting more than community support, thank you for your kind sharing of the links, i  will double check the system requirements, i think my issue is that i ran out of ressources because that causes usually the strnages problems.

The indexing meanhwile finished and found the new photos, for now i am happy, and i will see what will happen after i will resync photos from the mobile and how the server behaves.... 

thank you again, i am doing community and paid support for open source myself at the moment. i just use a different user there "adam-op" :)
